Docs/sphinx_doc/VerticalSCoord.rst

```diff
@@ -0,0 +1,93 @@
+
+Transformation Equation
+-----------------------
+
+.. math::
+   z\left( x,y,\sigma ,t\right) &=\zeta \left( x,y,t\right) +\left[ \zeta \left( x,y,t\right) +h\left( x,y\right) \right] S\left( x,y,\sigma \right) ,\\
+   S\left( x,y,\sigma \right) &=\frac{h_c \sigma +h \left( x,y\right) C\left( \sigma \right) }{h_c +h\left( x,y\right)}
+
+where :math:'S\left( x,y,\sigma \right)' is a nonlinear vertical transformation functional, :math:'\zeta \left( x,y,t\right)' is the time-varying free-surface, :math:'h\left( x,y\right) ' is the unperturbed water column thickness and :math:'z=-h\left( x,y\right) ' corresponds to the ocean bottom, :math:'\sigma ' is a fractional vertical stretching coordinate ranging from :math:'-1\leq \sigma \leq 0,C\left( \sigma \right) ' is a nondimensional, monotonic, vertical stretching function ranging from :math:'-1\leq C\left( \sigma \right) \leq 0', and :math:'h_c' is a positive thickness controlling the stretching.  In sediment applications, :math:'h=h\left( x,y,t\right) ' is changed at every time-step since it is affected by erosion and deposition processes.
+
+We find it convenient to define:
+
+.. math::
+   H_z \equiv \frac{\partial z}{\partial \sigma }
+
+where :math:'H_z = H_z \left( x,y,\sigma ,t\right) ' are the vertical grid thicknesses. In ROMSX, :math:'H_z' is computed discretely as :math:'\delta z/\delta \sigma ' since this leads to the vertical sum of :math:'H_z' being exactly the total water column thickness :math:'D'.
+
+Notice that,
+
+.. math::
+   S\left( x,y,\sigma \right) = \begin{cases}{0,& \text{if } \sigma = 0, & C\left( \sigma \right) = 0, & \text{at the free-surface;}}\\{-1 , & \text{if } \sigma = -1, & C\left( \sigma \right) = -1, & \text{at the ocean bottom}} \end{cases}
+
+This transformation offers several advantages:
+
+* Regardless of the design of :math:'C\left( \sigma \right) ', it behaves like equally-spaced sigma-coordinates in shallow regions, where :math:'h\left( x,y\right) \ll h_c'. This is advantageous because it avoids excessive resolution and associated CFL limitation in such areas.
+
+* Near-surface refinement behaves more or less like geopotential coordinates in deep regions (level thicknesses, :math:'H_z', do not depend or weakly depend on bathymetry), while near-bottom like sigma coordinates (:math:'H_z' is roughly proportional to depth). This reduces the extreme r-factors near the bottom and reduces pressure gradient errors.
+
+* The **true** sigma-coordinate system is recovered as :math:'h_c \rightarrow \infty '. This is useful when configuring applications with **flat** bathymetry and **uniform** level thickness. Practically, you can achieve this by setting Tcline to **1.0d+16** in roms.in. This will set :math:'h_c =1.0\times 10^{16}'. Although not necessary, we also recommend that you set :math:'\theta _S = 0' and :math:'\theta _B =0'.
```

Comment:
Tcline and `roms.in` should be updated to refer to ROMSX variables `tcline` and `inputs` (although the value of tcline is currently not read from the inputs file it does exist in the DataStruct file)
https://github.com/seahorce-scidac/ROMSX/blob/5b9432761548b09720f6ccc8c59cb54138a4bf1d/Source/DataStruct.H#L335-L338

Reply:
perfect, that was one of my known issues

Reply:
See if what it says now works?